external-crates/move/crates/move-unit-test/src/test_reporter.rs

```diff
@@ -68,11 +71,34 @@ pub struct TestResults {
     test_plan: TestPlan,
 }
 
+fn write_string_to_file(filepath: &str, content: &str) -> std::io::Result<()> {
+    let path = Path::new(filepath);
+    if let Some(parent) = path.parent() {
+        std::fs::create_dir_all(parent)?;
```

Comment:
Creating directories by default is a little bit strange in my opinion (unexpected to me that a tool would do that), and [`std::fs::write`](https://doc.rust-lang.org/std/fs/fn.write.html) already exists to perform the other part of this function.